bundle/manifests/patterns-operator.clusterserviceversion.yaml

```
@@ -74,6 +75,9 @@ spec:
           False'
         displayName: Use CSV
         path: gitOpsSpec.useCSV
+      - description: 'Unseal the Vault automatically. Note: This is *insecure*'
```

```suggestion
      - description: 'Manage the Vault automatically. Note: This is *insecure*'
```

because you also initialize it too

---

Ack, makes sense